plugins/ReactNativeOpenFile/setupBackend.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+var getData = require('../../backend/getData');
+var getDataFiber = require('../../backend/getDataFiber');
+
+import type Bridge from '../../agent/Bridge';
+import type Agent from '../../agent/Agent';
+
+module.exports = function setupRNOpenFile(bridge: Bridge, agent: Agent, openFileInEditor: () => void) {
+  bridge.on('rn-open-file', val => {
+    // '__REACT_DEVTOOLS_GLOBAL_HOOK__.$inst'
+    var el = val.split('.').reduce(
+      (prev, curr) => prev ? prev[curr] : null,
+      window
+    );
+    var inst = el._reactInternalInstance;
```

Comment:
> I would look at showComponentSource in Panel instead.

I tried `node.get('source')` just worked in `Panel`, obviously I did something unnecessarily. 😂

Also, I think it would be great if we just do [`launchEditor`](https://github.com/facebook/react-native/blob/master/local-cli/server/util/launchEditor.js) in Electron part, this will don't need go to RN process (call openFileInEditor) and support non-RN platforms, what do you think?